scripts/qa_crowbarsetup.sh

```diff
@@ -1535,7 +1535,7 @@ EOF
     if [ -z "$NOINSTALLCLOUDPATTERN" ] ; then
         safely $zypper in -l -t pattern cloud_admin
         # make sure to use packages from PTF repo (needs zypper dup)
-        $zypper mr -e cloud-ptf && safely $zypper dup --from cloud-ptf
+        $zypper mr -e cloud-ptf && safely $zypper -n dup --no-recommends --from cloud-ptf
```

Comment:
Thanks for the hint, seems like I wasn't careful enough during my rebase.
I removed the`-n`.